backend/elasticsearch/src/main/java/org/hibernate/search/backend/elasticsearch/scope/model/impl/ElasticsearchScopeModel.java

```diff
@@ -220,7 +221,7 @@ public String getNestedDocumentPath(String absoluteFieldPath) {
 				} )
 				.orElse( Optional.empty() );
 
-		return nestedDocumentPath.orElse( null );
+		return nestedDocumentPath.orElse( Collections.emptyList() );
```

Comment:
Same here. We can make the subsequent method return an empty list in case of no nested parent.